com.unity.probuilder/Documentation~/edit-mode-toolbar.md

```diff
@@ -5,23 +5,23 @@ Use the Edit Mode toolbar to switch between the four [ProBuilder Edit modes](mod
 ![The Edit mode toolbar](images/toolbar_editmode.png)
 
 * ![Object edit mode](images/icons/EditModes_Object.png) **Object** mode: the standard Unity mode for [selecting and manipulating GameObjects](object-actions.md).
-* ![Vertex edit mode](images/icons/EditModes_Vertex.png) **Vertex** mode: the element mode for [selecting and manipulating vertices](vertex.md) (points) on a ProBuilder Mesh
-* ![Edge edit mode](images/icons/EditModes_Edge.png) **Edge** mode: the element mode for [selecting and manipulating edges](edge.md) (lines) on a ProBuilder Mesh
-* ![Face edit mode](images/icons/EditModes_Face.png) **Face** mode: the element mode for [selecting and manipulating faces](face.md) (polygons) on a ProBuilder Mesh
+* ![Vertex edit mode](images/icons/EditModes_Vertex.png) **Vertex** mode: the element mode for [selecting and manipulating vertices](vertex.md) (points) on a ProBuilder Mesh.
+* ![Edge edit mode](images/icons/EditModes_Edge.png) **Edge** mode: the element mode for [selecting and manipulating edges](edge.md) (lines) on a ProBuilder Mesh.
+* ![Face edit mode](images/icons/EditModes_Face.png) **Face** mode: the element mode for [selecting and manipulating faces](face.md) (polygons) on a ProBuilder Mesh.
 
 The Vertex, Edge, and Face modes are also known collectively as the **Element** modes.
 
-> ***Tip:*** By default, the Edit mode toolbar appears in the top middle of the Scene view, but you can reposition it with the [Toolbar Location](preferences.md#toolbarloc) preference.
+> **Tip:** By default, the Edit mode toolbar appears in the top middle of the Scene view, but you can reposition it with the [Toolbar Location](preferences.md#toolbarloc) preference.
 
 
 
 ## Edit mode hotkeys
 
 In addition to the Scene view toolbar, [hotkeys](hotkeys.md) (keyboard shortcuts) are also available:
 
-- `Escape` switches to Object mode from any other mode.
-- `G` toggles between the Object and Element modes. For example, if you are in Vertex mode, pressing `G` once returns you to Object mode, but pressing `G` again switches back to Vertex mode again.
-- `H` toggles between the Element modes (Vertex, Edge, and Face).
+- `Escape` switches to **Object** mode from any other mode.
+- `G` toggles between the **Object** and **Element** modes. For example, if you are in **Vertex** mode, press `G` once to return to **Object** mode, and press it again to switch back to **Vertex** mode.
+- `H` toggles between the **Element** modes (**Vertex**, **Edge**, and **Face**).
 
 If you prefer to have dedicated keys for each mode, you can enable the [Unique Mode Shortcuts](hotkeys.md#unique_mode) preference.
```

Comment:
Starting at `4.0.4`, this is no longer a preference. We now support the Shortcut Manager editor, so the new way of setting dedicated keys for modes is to simply assign them in the Shortcut Manager window.